File: user-service/src/main/java/io/ecommerce/userservice/config/p6spy/P6spyLogMessageFormatConfiguration.java
package io.ecommerce.userservice.config.p6spy;

import com.p6spy.engine.spy.P6SpyOptions;
import org.springframework.context.annotation.Configuration;

import javax.annotation.PostConstruct;

/**
 * @author : choi-ys
 * @date : 2021/04/13 5:52 오후
 * @Content : p6spy 의존성을 통한 실행 쿼리 출력 MessageFormat설정
 */
@Configuration
public class P6spyLogMessageFormatConfiguration {

    @PostConstruct
    public void setLogMessageFormat() {
        P6SpyOptions.getActiveInstance().setLogMessageFormat(P6spySqlFormatConfiguration.class.getName());
    }
}

File: order-service/src/main/kotlin/io/ecommerce/orderservice/config/p6spy/P6spyLogMessageFormatConfiguration.kt
package io.ecommerce.orderservice.config.p6spy

import com.p6spy.engine.spy.P6SpyOptions
import javax.annotation.PostConstruct

/**
 * @author : choi-ys
 * @date : 2022/01/09 3:29 오후
 */
class P6spyLogMessageFormatConfiguration {
    @PostConstruct
    fun setLogMessageFormat() {
        P6SpyOptions.getActiveInstance().logMessageFormat = P6spySqlFormatConfiguration::class.java.name
    }
}